### ISSUE TITLE
Remove app.stub.js

### ISSUE THREAD

**Issue by lathonez** [lathonez/clicker#79]
This is the last horrible workaround we've got really, should pick it up with the Ionic guys to get to the bottom of it.

**Comment by lathonez** [comment]
Finally got an index.html wrapper to load in Chrome, surprisingly doesn't solve the problem

**Comment by lathonez** [comment]
Adding <ion-app> to Karma's static (`context.html`) solves it

```
<!DOCTYPE html>
<!--
This is the execution context.
Loaded within the iframe.
Reloaded before every execution run.
-->
<html>
<head>
  <title></title>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
</head>
<body>
  <!-- The scripts need to be in the body DOM element, as some test running frameworks need the body
       to have already been created so they can insert their magic into it. For example, if loaded
       before body, Angular Scenario test framework fails to find the body and crashes and burns in
       an epic manner. -->
  <script type="text/javascript">
    // sets window.__karma__ and overrides console and error handling
    // Use window.opener if this was opened by someone else - in a new window
    if (window.opener) {
      window.opener.karma.setupContext(window);
    } else {
      window.parent.karma.setupContext(window);
    }

    // All served files with the latest timestamps
    %MAPPINGS%
  </script>
  <!-- Dynamically replaced with <script> tags -->
  %SCRIPTS%
  <script type="text/javascript">
    window.__karma__.loaded();
  </script>
  <ion-app></ion-app>
</body>
</html>
```

which is _a lot_ nicer than this fuckery we're doing atm. Just need to figure out how to do that dynamically...

**Comment by lathonez** [comment]
https://github.com/karma-runner/karma/pull/1825

**Comment by lathonez** [comment]
That's all fine but obviously it hasn't actually been released yet.

**Comment by lathonez** [comment]
5de5e5d

Ready for when Karma release 0.13.23

**Comment by lathonez** [comment]
Need to update blog dependencies & files after this

**Comment by mavogel** [comment]
Awesome work man! :) 
Do you know when they'll release the new Karma version? Last release was in March. 
Somebody already asked in the [gitter-channerl](https://gitter.im/karma-runner/karma) but I guess there was no reply yet. Well I felt free to ask again ;)

FYI: I migrated you app to the latest beta6 of ionic because beta4 has an ugly bug and the app does not run on Android (whitescreen only). I hope they'll release Karma asap.

**Comment by lathonez** [comment]
Ah nice, hopefully someone gets back to you in gitter. Looking at the pattern of their last releases I feel like one should be due now!

I didn't know about that bug, will upgrade now..

**Comment by lathonez** [comment]
Removed as per https://github.com/lathonez/clicker/commit/5f096f7

The npm `postinstall` solution for karma static barely warrants a new version tbh. Have had to downgrade to `0.13.9` due to https://github.com/karma-runner/karma/issues/1788

**Comment by lathonez** [comment]
Re the previous comment, @lazaromenezes has implemented this correctly in #137
